posts/alpine-stack-size.md

```diff
@@ -0,0 +1,155 @@
+# An Alpine linux gotcha and what it means for you
+
+If you read Wikipedia you will find that Alpine is a linux distribution that is based on musl (more on this later) and BusyBox.
+
+With the rise of Docker, it has become a favoured distribution for Docker images due to the favourably small size of it's images.
+
+At the time of writing, the [Ruby docker images](https://hub.docker.com/_/ruby/tags/) are 343MB in their standard linux distribution format, but are 28MB in their Alpine flavour.
+
+## The tooling
+
+We use RSpec to test our code, a common testing framework for the Ruby language. 
+One of it's most powerful features is the ability to create "hierarchical test suites" by using context blocks
+
+As an example:
+
+```ruby
+describe 'some unit' do
+  context 'given something has happened' do
+    before { do_something }
+    
+    context 'when we perform some action' do
+      before { some_action }
+      
+      it 'has happened' do
+        expect(result).to have_happened
+      end
+    end
+  end
+end
+```
+
+These context blocks can get more powerful, and indeed allow sharing of setup between multiple test suites (can be desirable when controlling simulators of external services in tests).
```

Comment:
Same here you have powerful in the previous sentence and once more here. Replace by "compelling" or other synonym in the previous sentence?

Reply:
Resolved